services/catalog/src/simcore_service_catalog/models/schemas/services.py

```diff
@@ -132,3 +133,7 @@ class Config:
                 "url": "https://catalog:8080/services/a8f5a503-01d5-40bc-b416-f5b7cc5d1fa4",  # NEW self
             }
         }
+
+
+class ServiceResourcesGet(ServiceResources):
```

Comment:
here the inheritance is more clear than with the List before. Here we want to distinguish between these two classes even if they have the exact same fields

Reply:
I will use an alias as well. I used inheritance here originally to convert to camelcase since the RestAPI follows JS conventions.